docs/release-notes/cloud/cloud-2024-01-12-gh-repository-ssh-keys-discontinuation.md

```diff
@@ -0,0 +1,83 @@
+rss_href: /feed_rss_created.xml
+---
+
+# Discontinuation of SSH keys for GitHub repositories January 12, 2024
+
+!!! info "This release note applies only to GitHub"
+
+For increased security, Codacy is discontinuing the usage of **repository SSH keys** for Git operations on GitHub in favor of **installation access tokens**.
+
+This change translates into important security improvements:
+
+-   It limits the access level of the Codacy GitHub App, as it no longer requires [read and write repository permissions for Administration](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-administration).
+-   Contrary to long-living SSH keys, [installation access tokens](https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/generating-an-installation-access-token-for-a-github-app#about-installation-access-tokens) expire after one hour and Codacy loses access when the GitHub App is uninstalled.
+
+To minimize any impact, Codacy is currently discontinuing the usage of repository SSH keys but will keep using them as a fallback mechanism at this first stage. At a later stage, Codacy will [stop using repository SSH keys and delete the keys](#timeline) stored in our systems.
+
+## Changes on the Codacy GitHub App required permissions {: id="changes-permissions"}
+
+To use installation access tokens, the Codacy GitHub App now requires [repository read permissions for Contents](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-contents).
+
+On the other hand, the usage of installation access tokens no longer requires [read and write repository permissions for Administration](https://docs.github.com/en/rest/authentication/permissions-required-for-github-apps?apiVersion=2022-11-28#repository-permissions-for-administration). The Codacy GitHub App still requests this permission during this first stage, while using repository SSH keys as a fallback mechanism. At a later stage, Codacy will [remove the permission from the GitHub App](#timeline).
+
+## What do you need to do if you have a GitHub organization? {: id="to-do"}
+
+Make sure an organization owner [approves the updated permissions for the Codacy GitHub App](https://docs.github.com/en/apps/using-github-apps/reviewing-and-modifying-installed-github-apps) on your GitHub organization, if not done yet.
+
+Codacy has been requesting repository read permissions for **Contents** since September 2023. Organization owners should have received a GitHub notification to review a request for this additional permission:
+
+![Codacy GitHub App updated permissions request](../images/2023-12-DD-gh-updated-permissions.png)<!--TODO PLUTO-802 Rename image-->
+
+If you have any questions or need help, please contact <mailto:support@codacy.com>.
+
+## Removal of repository permissions for Administration and SSH keys {: id="timeline"}
```

Comment:
The release title says "Discontinuation of SSH keys for GitHub repositories January 12, 2024" - do you want to align the message?

Reply:
This section describes the timeline for the final removal of the permissions for Administration and SSH keys (as we have on other [deprecation releases](https://docs.codacy.com/release-notes/cloud/cloud-2023-10-13-bundler-audit-deprecation/)), but here the date is not defined.